bench-tps/tests/bench_tps.rs

```diff
@@ -7,14 +7,11 @@ use {
         cli::{Config, InstructionPaddingConfig},
         send_batch::generate_durable_nonce_accounts,
     },
-    solana_client::{
-        connection_cache::ConnectionCache,
-        tpu_client::{TpuClient, TpuClientConfig},
-    },
+    solana_client::tpu_client::{TpuClient, TpuClientConfig},
```

Comment:
ohhh this is good to know. i was actually going to ask about this, since `solana_client` seems to be just a wrapper around `solana_tpu_client`. sounds good!

Reply:
I am wondering if this needs to be separate PRs. Add in with dependency on `solana_client` and then another PR that switches over the dependencies to `solana_tpu_client`. A lot of the code including `bench-tps` relies on `solana_client`. For example, `bench-tps` implements `BenchTpsClient` for `solana_client/tpu_client` not `solana_tpu_client/tpu_client`. so switching this up in this PR as well may be large.

Reply:
If you want to do dependency updates in a separate PR, that's fine with me. I can definitely see the argument for that. Just let me know whether you think it makes more sense to do it before or after this PR.

Reply:
I think it makes more sense to do it after this PR. This code will be following same usage as previous code (aka using `solana_client::tpu_client`. I have another PR in drafts that will then switch over to using `solana-tpu-client`. Then I will make another PR that finally switches over `ThinClient` in `LocalCluster` to `solana-tpu-client`.